ansible/roles/contiv/tasks/netplugin.yml

```diff
@@ -0,0 +1,81 @@
+- name: ensure pluginDir exists
+  file: path=/usr/libexec/kubernetes/kubelet-plugins/net/exec recurse=yes state=directory
+
+- name: copy contiv cni conf file
+  copy: src=contiv_cni.conf dest=/usr/libexec/kubernetes/kubelet-plugins/net/exec/contiv_cni.conf
+
+- name: ensure cni bin dir exists
+  file: path=/opt/cni/bin recurse=yes state=directory
+
+- name: copy contiv cni exec file
+  copy: src={{ contiv_bin_path }}/contivk8s dest=/opt/cni/bin/contivk8s.bin mode=0755
+
+- name: ensure contiv config directory exists
+  file: path=/opt/contiv/config recurse=yes state=directory
+
+- name: setup config for the contiv cni plugin
+  template: src=contiv.cfg.j2 dest=/opt/contiv/config/contiv.json
+
+- name: copy netplugin exec file
+  copy: src={{ contiv_bin_path }}/netplugin dest=/usr/bin/netplugin mode=0755
+
+- name: get openstack kilo rpm
```

Comment:
I don't really know the details of all CoreOS issues. If you're aware of the issue, why don't you open it?